edit/location/location.go

```diff
@@ -31,17 +31,22 @@ type mode struct {
 	binding eddefs.BindingMap
 	hidden  vector.Vector
 	pinned  vector.Vector
+	matcher eval.Callable
 }
 
+var matchPattern = eval.NewBuiltinFn("edit:location:match-pattern", patternMatch)
```

Comment:
I would prefer `match-dir-pattern`.

Don't name one thing `AB` and another `BA` just to avoid name conflict :) You can call the function `matchDirPattern` and the variable `matchDirPatternBuiltin`

Reply:
Done.